features/features.go

```diff
@@ -31,8 +30,7 @@ const (
 
 // List of features and their default value, protected by fMu
 var features = map[FeatureFlag]bool{
-	unused:                   false,
-	IDNASupport:              false,
+	unused: false,
```

Comment:
Not sure why the formatting isn't catching this but can you run `gofmt` on this file (or just manually align this with everything else in the map).